root/static/scripts/common/components/DescriptiveLink.js

```diff
@@ -14,19 +14,21 @@ import ArtistCreditLink from './ArtistCreditLink';
 import EntityLink from './EntityLink';
 
 type DescriptiveLinkProps = {
+  +allowNew?: boolean,
   +content?: React.Node,
   +entity: CoreEntityT,
   +showDeletedArtists?: boolean,
   +target?: '_blank',
 };
 
 const DescriptiveLink = ({
+  allowNew,
   content,
   entity,
   showDeletedArtists = true,
   target,
 }: DescriptiveLinkProps) => {
-  const props = {content, showDisambiguation: true, target};
+  const props = {content, showDisambiguation: true, target, allowNew};
```

Comment:
You're not actually using allowNew anywhere, as far as I can see, so what's the point of it? :)

Reply:
I'm using allowNew in AddPlace to ensure the place can be displayed with gid being present in the database